zio-kafka-testkit/src/main/scala/zio/kafka/testkit/KafkaTestUtils.scala

```diff
@@ -92,6 +93,19 @@ object KafkaTestUtils {
         Serde.string
       )
 
+  /**
+   * A stream that produces messages to a Topic on a schedule for as long as it is running.
+   */
+  def scheduledProducer[R](
```

Comment:
```suggestion
  def scheduledProduce[R](
```

as it's not a Producer but a producing action?